### PR TITLE
CRS-516 Add external dns annotations for preprod - as advised by CP team

### PR DESCRIPTION
**Pull request by yasinmustafa**

### CODE REVIEW

File: helm_deploy/values-preprod.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ generic-service:
 
   ingress:
     host: calculate-release-dates-preprod.hmpps.service.justice.gov.uk
+    contextColour: green
```
Comment (petergphillips) [inline helm_deploy/values-preprod.yaml:9]: would be better to set the default as `green` in `values.yaml` and then override for dev, but guess it depends when you're going to production and what cluster that will be in...

Comment (petergphillips) [inline helm_deploy/values-preprod.yaml:9]: would be better to set the default as `green` in `values.yaml` and then override for dev, but guess it depends when you're going to production and what cluster that will be in...

**Review by petergphillips — APPROVED** [review]
LGTM

File: helm_deploy/values-preprod.yaml (hunk 1 above)
Comment (yasinmustafa) [reply helm_deploy/values-preprod.yaml:9]: I think that makes sense. So the next step will be to create the prod set up and that will also be on the live cluster. Then after that dev will be migrated

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**